app/assets/stylesheets/application.scss

```diff
@@ -1,11 +1,14 @@
 @charset "utf-8";
 
 /*
+
  * = require us_web_design_standards
  * = require us_web_design_standards_fonts
+ * = require swagger-ui/screen
  */
 
  @import "variables";
+ @import "reset";
```

Comment:
it seems like this css file is specific to swagger styles -- maybe name it more specifically for clarity?